autopush-common/src/errors.rs

```diff
@@ -155,6 +155,8 @@ pub enum ApcErrorKind {
 
     #[error("Endpoint Error: [{0}] {1}")]
     EndpointError(&'static str, String),
+    #[error("Upstream Error: [{0}] {1}")]
+    UpstreamError(String, String),
```

Comment:
Echoing my other comment I don't think this is needed since this is endpoint specific and not triggered by any common code.